logstash-core/src/main/java/org/logstash/common/io/RecordIOReader.java

```diff
@@ -46,6 +49,7 @@
     private final Path path;
     private long channelPosition;
     private static final int UNSET = -1;
+    private static final Logger logger = LogManager.getLogger(RecordIOReader.class);
```

Comment:
Once you incorporate the file's path into the exception's message, the `Logger` instance will no longer be necessary.

Reply:
yes. removed it.